src/cli/onefuzz/backend.py

```diff
@@ -45,16 +45,24 @@ def _temporary_umask(new_umask: int) -> Generator[None, None, None]:
             os.umask(prev_umask)
 
 
+class BackendConfig(BaseModel):
+    authority: str
+    client_id: str
+    client_secret: Optional[str]
+    endpoint: Optional[str]
+    features: Dict[str, bool] = Field(default_factory=dict)
```

Comment:
a set to represent the list of enabled feature might be easier to read when dumping the config with onefuzz config

Reply:
Good idea.

I was thinking about this from underlying JSON, which doesn't directly support `set`.    However, pydantic maps `Set[str]` to `List[str]` with appropriate checking/deduplication.